pom.xml

```diff
@@ -144,6 +144,13 @@
 			<artifactId>spring-oxm</artifactId>
 		</dependency>
 		<!-- Spring end -->
+		<!-- Override start -->
+		<dependency>
+			<groupId>org.apache.tika</groupId>
+			<artifactId>tika-parser-code-module</artifactId>
```

Comment:
The package `tika-parsers-standard-package` has already included the latest Tika version. I don't think this override is necessary:
```
[INFO] +- org.apache.tika:tika-parsers-standard-package:jar:3.1.0:compile
[INFO] |  +- org.apache.tika:tika-parser-apple-module:jar:3.1.0:compile
[INFO] |  |  +- org.apache.tika:tika-parser-zip-commons:jar:3.1.0:compile
[INFO] |  |  \- com.googlecode.plist:dd-plist:jar:1.28:compile
[INFO] |  +- org.apache.tika:tika-parser-audiovideo-module:jar:3.1.0:compile
[INFO] |  +- org.apache.tika:tika-parser-cad-module:jar:3.1.0:compile
[INFO] |  +- org.apache.tika:tika-parser-code-module:jar:3.1.0:compile
```